src/Apps/Sell/Mutations/useUpdateMyCollectionArtwork.tsx

```diff
@@ -0,0 +1,30 @@
+import { useMutation } from "Utils/Hooks/useMutation"
+import { graphql } from "react-relay"
+import { useUpdateMyCollectionArtworkMutation } from "__generated__/useUpdateMyCollectionArtworkMutation.graphql"
+
+export const useUpdateMyCollectionArtwork = () => {
+  return useMutation<useUpdateMyCollectionArtworkMutation>({
+    mutation: graphql`
+      mutation useUpdateMyCollectionArtworkMutation(
+        $input: MyCollectionUpdateArtworkInput!
+      ) {
+        myCollectionUpdateArtwork(input: $input) {
+          artworkOrError {
+            ... on MyCollectionArtworkMutationSuccess {
+              artwork {
+                title
+                internalID
+              }
```

Comment:
What do you think about returning images here as well?

Reply:
Do we need images here?

Reply:
I was wondering, if the user added new images, went to see the updated artwork in my collection and the images are still processing, what would we show?